### PR TITLE
New h5 files and 2.3 database migration

### PR DESCRIPTION
**Pull request by abeasley1722**

### CODE REVIEW

File: tools/DatabaseMigration/2_3/2_3_DatabaseMigration.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#2_1_DatabaseMigration.py
+#----------------------------------
+# Created By: Anointiyae Beasley
+# Created Date: 06/18/2024
+# Version 1.0
+#----------------------------------
+"""This is a database migration script that will initialize version
+    2.1 of the database. The change from version 2.0 to 2.1 is that
```
Comment (lovelysandlonelys) [inline tools/DatabaseMigration/2_3/2_3_DatabaseMigration.py:9]: I always forget to change the file descriptions too. At the top it should be 2_3 database migration and we should be moving from version 2.2 to 2.3 where we're adding a datum.

**Review by lovelysandlonelys — APPROVED** [review]
Looks fantastic joy! Just one documentation comment.

File: tools/DatabaseMigration/2_3/2_3_DatabaseMigration.py (hunk 1 above)
Comment (abeasley1722) [reply tools/DatabaseMigration/2_3/2_3_DatabaseMigration.py:9]: Thanks!

**Review by matdenkas — APPROVED** [review]
LGTM

File: tools/DatabaseMigration/2_3/2_3_DatabaseMigration.py (hunk 1 above)
Comment (matdenkas) [reply tools/DatabaseMigration/2_3/2_3_DatabaseMigration.py]: Nice catch!

**Pull request merged**